docs/tools/vdb_table/data/meilisearch.json

```diff
@@ -47,34 +47,34 @@
     "comment": "https://www.meilisearch.com/docs/learn/fine_tuning_results/geosearch"
   },
   "multi_vec": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Up to 65536 vectors by document"
   },
   "sparse_vectors": {
     "support": "none",
     "source_url": "",
     "comment": ""
   },
   "bm25": {
-    "support": "none",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://github.com/orgs/meilisearch/discussions/674",
+    "comment": "Support scoring but not exactly BM25"
   },
   "full_text": {
     "support": "full",
     "source_url": "",
     "comment": ""
   },
   "embeddings_text": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://www.meilisearch.com/docs/learn/experimental/vector_search",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://www.meilisearch.com/docs/learn/experimental/vector_search",
+    "comment": "Custom vectors can be linked to documents and provided during search"
```

Comment:
this should have been reverted based on https://github.com/superlinked/VectorHub/issues/210#issuecomment-1951425832